changelog/13365.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:feature
+**Add PKCE support to OIDC Auth**: The Authorization Code flow makes use of the
```

Comment:
I think you'll need to cherry-pick the changelog modification commit too.

Reply:
I was going to create another PR for that. But that makes sense to do it all in this PR.